backend/controllers/createAuthUser.js

```diff
@@ -54,11 +56,20 @@ const createAuthUserFactory = (
         })
       } catch (err) {
         if (err.status === 400 && err.response && err.response.body) {
-          const { emailError: error, error_description: errorDescription, field } = err.response.body
+          const { emailError: error, userMessage: errorDescription } = err.response.body
+
+          const updateDes = errorDescription
+            .replace('Validation failure:', '')
+            .split(';')
+            .map((x) => x.split(':'))
+          const eachFieldIdWithError = updateDes[0].toString().split(',')
+
           const description =
-            error === 'email.domain' ? 'The email domain is not allowed.  Enter a work email address' : errorDescription
+            error === 'email.domain'
+              ? 'The email domain is not allowed.  Enter a work email address'
+              : eachFieldIdWithError[1]
 
-          const errorDetails = [{ href: `#${field}`, text: description }]
+          const errorDetails = [{ href: `#${eachFieldIdWithError[0].toString().trim()}`, text: description }]
```

Comment:
I thought we had changes this so it will only return a single error at a time?   or does the new endpoint not do that in manage-users-api ?

Reply:
now multiple errors can be sent in new endpoint because of model validation at field level